learn/inner_workings/datatypes.mdx

```diff
@@ -12,14 +12,18 @@ String tokenization is the process of **splitting a string into a list of indivi
 
 A string is passed to a tokenizer and is then broken into separate string tokens. A token is a **word**.
 
-- For Latin-based languages, the words are separated by **space**.
-- For Kanji characters, the words are separated by **character**.
+- For Latin-based languages, the words are separated by **space**
+- For Kanji characters, the words are separated by **character**
```

Comment:
It's wrong, Kanjis are no more split by character, but the words are split based on Language dictionaries depending on how well the Language is supported in Meilisearch.